### PR TITLE
Enable receipt processing for mainnet

### PR DESCRIPTION
**Pull request by rlan35**

### CODE REVIEW

File: node/node.go [hunk 1]
```diff
@@ -290,12 +290,10 @@ func (node *Node) AddPendingTransaction(newTx *types.Transaction) {
...
-	if node.NodeConfig.GetNetworkType() != nodeconfig.Mainnet {
-		node.pendingCXMutex.Lock()
-		node.pendingCXReceipts = append(node.pendingCXReceipts, receipts)
-		node.pendingCXMutex.Unlock()
-		utils.Logger().Error().Int("totalPendingReceipts", len(node.pendingCXReceipts)).Msg("Got ONE more receipt message")
-	}
+	node.pendingCXMutex.Lock()
+	node.pendingCXReceipts = append(node.pendingCXReceipts, receipts)
+	node.pendingCXMutex.Unlock()
+	utils.Logger().Error().Int("totalPendingReceipts", len(node.pendingCXReceipts)).Msg("Got ONE more receipt message")
```
Comment (LeoHChen) [inline node/node.go]: Please change Error to Info/Debug.

Comment (rlan35) [reply node/node.go]: Done.

**Comment by rlan35** [comment]
Jenkins, test this please.

**Comment by rlan35** [comment]
Jenkins, test this please.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

**Pull request merged**